packages/benchmarks/src/makeRadiantChatLlm.ts

```diff
@@ -15,14 +16,14 @@ export const makeRadiantChatLlm = async ({
   deployment: string;
   mongoDbAuthCookie?: string;
   lmmConfigOptions: Pick<
-    OpenAI.default.Chat.ChatCompletionCreateParams,
+    OpenAI.Chat.ChatCompletionCreateParams,
```

Comment:
I don't dislike the namespaced types here (very explicit what it is!). Is it _possible_ to import just `ChatCompletionCreateParams` without the whole `OpenAI` namespace?

Reply:
i dont think so. dont think it's a real issue tho b/c the benchmarks package is only being built in this repo, which will already install all of `mongodb-rag-core`, which is dependent on the `openai` package